corehq/apps/export/models/new.py

```diff
@@ -792,6 +800,76 @@ def _new_from_schema(cls, schema):
         )
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = SMS_EXPORT
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        return cls(
```

Comment:
this is interesting. yeah this does feel a lot like "static" generation vs. "dynamic" generation. a json structure somewhere would make sense to me